04-Multi-Band-Raster.Rmd

```diff
@@ -117,104 +126,92 @@ par(original_par) # go back to original par
 
 ```
 
-#Other Types of Multi-band Raster data
+#Other Types of Multi-band Raster Data
 
 Multi-band raster data might also contain:
 
-1. Time series - of the same variable, over the same area over time.
-2. Multi-hyperspectral imagery - that might have 4 or more bands up to 400+ bands 
+1. Time series: same variable, over the same area, over time.
+2. Multi-hyperspectral imagery: that might have 4 or more bands up to 400+ bands 
 of image data!
 
-NOTE: In a multi-band dataset, the rasters will always have the same `extent`,
-`CRS` and `resolution`.
-{: .notice }
+>NOTE: In a multi-band dataset, the rasters will always have the same extent,
+>CRS and resolution.
 
 In this lesson, the multi-band data that we are working with is imagery
-collected using the NEON Airborne Observation Platform high resolution camera over
+collected using the <a href="http://http://www.neoninc.org/science-design/collection-methods/airborne-remote-sensing" target="_blank">NEON Airborne Observation Platform</a> high resolution camera over
 the <a href="http://www.neoninc.org/science-design/field-sites/harvard-forest" target="_blank">NEON Harvard Forest field site</a>. 
+These RGB images are 3-band rasters. Most of what we will do can be transferred 
+to rasters with a different number of bands.
 
-
-#Getting Started with Multi-Band data
+#Getting Started with Multi-Band Data
 If we read a raster stack into R using the `raster` function, it defaults to 
-reading in one (the first) band. We can plot this band using the plot function.
-
-In a typical GIS application, a single band would render a single image 
```

Comment:
I moved this section earlier in the lesson when we were looking at Multi-band Imagery (before starting to teach the coding).  I felt it fit better there as it was the first time we were seeing the gray-scale images.